### ISSUE TITLE
CI: fix pypy3

### ISSUE THREAD

**Issue by oberstet** [crossbario/autobahn-python#1335]
https://travis-ci.org/crossbario/autobahn-python/builds/656986060

**Comment by meejah** [comment]
Hmm, works locally for me .. I have a much newer pypy3 version that what's on travis, though:

```
Python 3.5.3 (7.0.0+dfsg-3, Feb 21 2019, 03:51:22)
[PyPy 7.0.0 with GCC 8.2.0]
```

**Comment by oberstet** [comment]
^ mmh, yeah, pypy is at PyPy3.6 v7.3.0. any idea how to use that on travis? hopefully that would make the issue disappear;)

**Comment by meejah** [comment]
Hmm, this says it should be v7.1.1already https://docs.travis-ci.com/user/languages/python/#pypy-support

**Comment by oberstet** [comment]
^ ah, great! maybe it would then just work bumping to the newer version in travis config ..

**Comment by oberstet** [comment]
long ago fixed: works on pypy-3.8, see eg https://github.com/crossbario/autobahn-python/runs/5954123054?check_suite_focus=true
